cabal-install/src/Distribution/Client/Init/Types.hs

```diff
@@ -455,7 +455,7 @@ newtype BreakException = BreakException String deriving (Eq, Show)
 instance Exception BreakException
 
 -- | Used to inform the intent of prompted messages.
-data Severity = Log | Info | Warning | Error deriving (Eq, Show)
+data Severity = Info | Warning | Error deriving (Eq, Show)
```

Comment:
Why do we have separate log levels (severities? are they terms interchangeable here?) for `Init`? 
If we don't want to use the ones in `Distribution.Verbosity` because that's in Cabal, we could have them somewhere in `Distribution.Client`

Reply:
Thank you for the food for thought.

Reply:
@andreabedini these logs are separate because it's good UX to have by default - it's not clear what is and is not actionable logging in the output from stdout without some form of loglevel. 

I'm amenable to following up with a PR that improves this by processing the verbosity to pick and choose the loglevel, but I wouldn't want them gone completely.

Reply:
Ok, that's reasonable, we had to introduce separate log levels that have a specific meaning to this phase :+1: